content/en/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md

```diff
@@ -60,7 +60,7 @@ with maintainers to let them know about this change.
 The following diagram shows who's affected by this change in a visual form
 (click on diagram for the larger version):
```

Comment:
Try this instead:
```suggestion
_(click on diagram for the larger version)_
```

Reply:
Oops, I should have read that, thank you! Will close this PR for now.